### PR TITLE
DOC: added all projects toctree to databroker

### PR DESCRIPTION
**Pull request by mrakitin**
## Description

Add all projects toctree to the databroker documentation.

## Motivation and Context

Motivated by https://github.com/NSLS-II/docs/issues/73.

## How Has This Been Tested?

Tested locally by building the documentation.

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
this looks good to me (and I'm trusting that it builds when you say it does)
@danielballan is this good to go?

**Comment by danielballan** [comment]
Let's get Travis's BUILD_DOCS build to pass before we merge. I just kicked it to re-run. Merge at will if/when it passes.

**Comment by mrakitin** [comment]
Probably it needs rebasing and manual resolution of the conflict.

**Comment by danielballan** [comment]
I think we should revisit whether and how to do this. The number of packages is growing large. Instead of linking to them all, maybe just link to blueskyproject.io? What do you think, @mrakitin?

**Comment by mrakitin** [comment]
I agree with that approach.

**Comment by mrakitin** [comment]
I rebased on master and changed all the project documentation links to a single URL. Does it make sense, @danielballan?

**Comment by danielballan** [comment]
Thanks! Do you think we should remove the NSLS-II link as well, just to give the right impression? And maybe replace BugReports with the gitter, which is probably a better first place to go for help?

**Comment by mrakitin** [comment]
It's available via 560185a.

File: doc/source/index.rst [hunk 1]
```diff
@@ -23,3 +23,16 @@ Index
...
+   :caption: Bluesky Project
+
+   Bluesky Project <https://blueskyproject.io>
+   Bluesky Project Repositories <https://github.com/bluesky>
+
+.. toctree::
+   :hidden:
+   :caption: Getting Help
+
+   Gitter <https://gitter.im/NSLS-II/DAMA>
```
Comment (danielballan) [inline doc/source/index.rst:38]: Actually --- In the spirit of scrubbing facility-specific branding out, should we move to a bluesky gitter? The way projects do this (e.g. jupyterlab moved from jupyter/jupyterlab -> jupyterlab/jupyterlab) is to post a big message in the gitter with a link to the new one. There is no way to set up an actual redirect.

Comment (danielballan) [reply doc/source/index.rst:38]: If this were _mainly_ an NSLS-II support channel I would say we should leave it as is, but it seems that a fair fraction of the questions we get are from other facilities that are evaluating bluesky.

Comment (stuartcampbell) [reply doc/source/index.rst:38]: I agree

**Comment by danielballan** [comment]
Good enough for me. We might want to resolve the gitter question before we update _all_ the repos though, to save effort.

**Pull request merged**